### ISSUE TITLE
Complete 2 surveys at once -> Crash

### ISSUE THREAD

**Issue by ivanarrizabalaga** [EyeSeeTea/malariapp#964]

**Comment by ivanarrizabalaga** [comment]
@ifoche I'm not able to avoid a conflict while trying to push the control dataelements for dates (CreatedOn, UpdatedOn).

According to the docs:
https://www.dhis2.org/download/apidocs/org/hisp/dhis/system/util/DateUtils.html

it should be the sort format but I'm still getting this

```json
{
	"httpStatus": "Conflict",
	"httpStatusCode": 409,
	"status": "WARNING",
	"message": "One more conflicts encountered, please check import summary.",
	"response": {
		"responseType": "ImportSummaries",
		"imported": 24,
		"updated": 0,
		"deleted": 0,
		"ignored": 1,
		"importSummaries": [{
			"responseType": "ImportSummary",
			"status": "SUCCESS",
			"importCount": {
				"imported": 24,
				"updated": 0,
				"ignored": 1,
				"deleted": 0
			},
			"conflicts": [{
				"object": "Jb72CVvIQEM",
				"value": "value_not_valid_datetime"
			}],
			"reference": "n5K3hMCnkax",
			"href": "https://eds-dev.psi-mis.org/api/events/n5K3hMCnkax"
		}]
	}
}
```

Would you mind to change it again into a Date to check it out tomorrow ?
PD:I've tried several different formats (GMT, long date, and so but none is working)

**Comment by ifoche** [comment]
Hi @ivanarrizabalaga. They're both Date now. It's strange but even if we changed it in our last meeting (I think I even was sharing my screen when I did it), one of them (_CreatedOn) was still in Date format...sorry about that, the only explanation that I find for that is that I didn't apply the change to the DE. I was about to tell you to retry with both Date&Time, but looking at your error it says "value_not_valid_datetime"... so the one that had Date&Time type didn't accepted the format...and nothing would change if I put both in Date&Time. 

In a nutshell, I've changed both to Date, sorry for the one that was not changed, and please let me know if you find any further problem with the control DEs
